drivers/dma/dma_intel_adsp_gpdma.c

```diff
@@ -203,7 +204,15 @@ static int intel_adsp_gpdma_start(const struct device *dev, uint32_t channel)
 
 static int intel_adsp_gpdma_stop(const struct device *dev, uint32_t channel)
 {
-	int ret = dw_dma_stop(dev, channel);
+	struct intel_adsp_gpdma_data *dev_data = dev->data;
+	int ret;
+
+	if (!dev_data->enabled) {
```

Comment:
in case of `CONFIG_PM_DEVICE=n` we will never stop the DMA?

Reply:
@ujfalusi Right, marking current version with DNM. This needs to be fixed.